src/Services/Bookings/Booking.API/Logic/BookingLogic.cs

```diff
@@ -82,5 +83,15 @@ private bool CheckCollision(Booking first, Booking second)
             }
             else return false;
         }
+
+        public async Task<IEnumerable<Booking>> GetBookingsAsync()
```

Comment:
You added public methods after private. 
Read 
https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods